project-p4-chatbot-master/code/script.js

```diff
@@ -0,0 +1,182 @@
+    chat.innerHTML += `
+      <section class="bot-msg">
+        <img src="assets/bot.png" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `
+  }
+
+// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  chat.scrollTop = chat.scrollHeight
+}
+
+// Question Sequence
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => foodOptions(message), 1000)
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => ambianceOptions(message), 1000)
+  } else if (questionNumber === 3) {
+    userReply(message)
+    setTimeout(() => showRecommendation(message), 1000)
+  } else {
+    userReply(message)
+    setTimeout(thankYou, 1000)
+  }
+}
+
+// Starts here
+const greeting = () => {
+  questionNumber = 1
+  botReply(`Welcome! What's your name?`)
+}
+
+const foodOptions = (msg) => {
+  questionNumber++
+  botReply(
+    `Hello ${msg}! What are you in the mood to eat today?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="pizzaBtn">Pizza</button>
+    <button id="mexicanBtn">Mexican</button>
+    <button id="sushiBtn">Sushi</button>
+  `
+  document
+  .getElementById('pizzaBtn')
+  .addEventListener('click', () => nextQuestion('Pizza'))
+  document
+  .getElementById('mexicanBtn')
+  .addEventListener('click', () => nextQuestion('Mexican'))
+  document
+  .getElementById('sushiBtn')
+  .addEventListener('click', () => nextQuestion('Sushi'))
+}
+
+const ambianceOptions = (type) => {
+  questionNumber++
+
+  botReply(`${type}, sounds yummy! What ambiance are you in the mood for?`)
+  
+  if (type === 'pizza') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Ambiance Options</option>
+        <option value="quickLunch">Quick Lunch</option>
+        <option value="dateNight">Date Night</option>
+        <option value="outWithFriends">Out With Friends</option>
+      </select>
+    `
+  } else if (type === 'mexican') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Ambiance Options</option>
+        <option value="quickLunch">Quick Lunch</option>
+        <option value="dateNight">Date Night</option>
+        <option value="outWithFriends">Out With Friends</option>
+      </select>
+    `
+  } else {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Ambiance Options</option>
+        <option value="quickLunch">Quick Lunch</option>
+        <option value="dateNight">Date Night</option>
+        <option value="outWithFriends">Out With Friends</option>
+      </select>
+    `
+  }
+  
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => nextQuestion(select.value))
```

Comment:
Here you are sending the `select.value` as a param to the next question so to get the `"Out with friends"` instead of `outWithFriends` I would suggest that you change the value to that:
`<option value="Out with friends">Out With Friends</option>`